language/tools/move-package/src/resolution/dependency_graph.rs

```diff
@@ -711,16 +973,67 @@ impl DependencyGraph {
         let manifest = parse_move_manifest_from_file(&pkg_path)
             .with_context(|| format!("Parsing manifest for '{}'", name))?;
 
+        let inserted_pkg = pkg.clone();
         let kind = pkg.kind.clone();
         self.extend_graph(
             &kind,
             &manifest,
             &pkg_path,
             dependency_cache,
             external_requests,
+            overrides,
             progress_output,
         )
-        .with_context(|| format!("Resolving dependencies for package '{}'", name))
+        .with_context(|| format!("Resolving dependencies for package '{}'", name))?;
+        Ok(inserted_pkg)
+    }
+
+    fn override_verify(
+        &self,
+        pkg: &Package,
+        name: PM::PackageName,
+        overrides: &BTreeMap<PM::PackageName, Package>,
+    ) -> Result<()> {
+        // check if any (should be 0 or 1) edges are the overrides of pkg
+        let pkg_overrides: Vec<_> = self
+            .package_graph
+            .neighbors_directed(name, Direction::Incoming)
```

Comment:
nit: use `edges_directed` to avoid the lookup on `edge_weight` and the `unwrap`.

Reply:
I was trying to find a better fitting function, but I don't think that `edges_directed` is implemented for `GraphMap` (nor is the `IntoEdgesDirected`)

Reply:
I seem to see it here, in case I'm missing something: 

https://docs.rs/petgraph/latest/petgraph/graphmap/struct.GraphMap.html#method.edges_directed